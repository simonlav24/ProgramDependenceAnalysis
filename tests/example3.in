# Example 3: everything is from the same register:
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
1 1 1 1
