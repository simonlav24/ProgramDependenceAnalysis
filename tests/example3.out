 # ./dflow_calc example3.in p0 p1 p2 p3 p4 p5 p6 p7 p8 p9 p10 p11 p12 p13 p14 d0 d1 d2 d3 d4 d5 d6 d7 d8 d9 d10
Reading the program file example3.in ... Found 17 instructions
getDepDepth(0)==0
getDepDepth(1)==1
getDepDepth(2)==2
getDepDepth(3)==3
getDepDepth(4)==4
getDepDepth(5)==5
getDepDepth(6)==6
getDepDepth(7)==7
getDepDepth(8)==8
getDepDepth(9)==9
getDepDepth(10)==10
getDepDepth(11)==11
getDepDepth(12)==12
getDepDepth(13)==13
getDepDepth(14)==14
getInstUsers(1)==1: 2  
getInstUsers(2)==1: 3  
getInstUsers(3)==1: 4  
getInstUsers(4)==1: 5  
getInstUsers(5)==1: 6  
getInstUsers(6)==1: 7  
getInstUsers(7)==1: 8  
getInstUsers(8)==1: 9  
getInstUsers(9)==1: 10  
getInstUsers(10)==1: 11  
getInstUsers(11)==1: 12  
getInstUsers(12)==1: 13  
getInstUsers(13)==1: 14  
getInstDeps(0)=={-1,-1}
getInstDeps(1)=={0,0}
getInstDeps(2)=={1,1}
getInstDeps(3)=={2,2}
getInstDeps(4)=={3,3}
getInstDeps(5)=={4,4}
getInstDeps(6)=={5,5}
getInstDeps(7)=={6,6}
getInstDeps(8)=={7,7}
getInstDeps(9)=={8,8}
getInstDeps(10)=={9,9}
